src/main/java/eu/dissco/orchestration/backend/repository/MappingRepository.java

```diff
@@ -47,28 +48,53 @@ public MappingRecord getMapping(String id) {
         .fetchOne(this::mapToMappingRecord);
   }
 
+  public MappingRecord getMappingOmitDeleted(String id) {
```

Comment:
I would just call this `getMapping` or if you want `getActiveMapping` this seems a bit long. Or add get `latest` in it.

Reply:
`getActiveMapping` is most accurate. 
It needs to be distinct from `getMapping`, which may or may not return a deleted mapping (as we discussed, we might want to do this if the user specifically requests it.)